test.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="test.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div>
+        <p class="score">0    :    0</p>

```

Comment:
✨P2: 스코어 보드판에서 \<p> 태그를 3개로 분리하여 사용할 것을 권장합니다. js 값에서 접근할 때도, 수정할 때도 그게 더 편리할 것 같아요! 아래 예시 코드를 추가합니다. 

```suggestion
           <div class="score-board">
      <p class="user-score">0</p>
      <p class="vs">:</p>
      <p class="computer-score">0</p>
    </div>

```